src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.hpp

```diff
@@ -1710,7 +1710,7 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::DoInit(otInstanc
     VerifyOrExit(otErr == OT_ERROR_NONE, err = MapOpenThreadError(otErr));
 
     // Enable automatic assignment of Thread advertised addresses.
-#if OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE
+#if defined(OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE)
```

Comment:
```suggestion
#if defined(OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE) && OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE
```
This can be set to 0 as well.

Reply:
fixed